app/services/courses/age_range_calculator_service.rb

```diff
@@ -0,0 +1,35 @@
+class Courses::AgeRangeCalculatorService
+  def self.determine_age_range_from(course)
+    new().calculate(course)
+  end
+
+  def calculate(course)
+    return calculate_using_level(course) if course.age_range_in_years.nil?
```

Comment:
Sorry for being a little nit-picky, but I don't think this expresses well as a guard.

Could we use an if-else statement here to make the core logic here clearer?

Reply:
Also, come to think of it, this logic is changed in the ticket, so we need to update the code to reflect that.